src/c/project-contribution-report-content-card.js

```diff
@@ -5,96 +5,148 @@ import h from '../h';
 const projectContributionReportContentCard = {
     controller(args) {
         const project = args.project(),
-              stateClass = (state) => {
-                  const classes = {
-                      online: {
-                          'paid': 'text-success.fa-circle',
-                          'refunded': 'text-error.fa-circle',
-                          'pending_refund': 'text-error.fa-circle',
-                          'pending': 'text-waiting.fa-circle',
-                          'refused': 'text-error.fa-circle'
-                      },
-                      failed: {
-                          'paid': 'text-error.fa-circle-o',
-                          'refunded': 'text-refunded.fa-circle',
-                          'pending_refund': 'text-refunded.fa-circle-o',
-                          'pending': 'text-refunded',
-                          'refused': 'text-refunded'
-                      },
-                      waiting_funds: {
-                          'paid': 'text-success.fa-circle',
-                          'refunded': 'text-error.fa-circle',
-                          'pending_refund': 'text-error.fa-circle',
-                          'pending': 'text-waiting.fa-circle',
-                          'refused': 'text-error.fa-circle'
-                      },
-                      successful: {
-                          'paid': 'text-success.fa-circle',
-                          'refunded': 'text-error.fa-circle',
-                          'pending_refund': 'text-error.fa-circle',
-                          'pending': 'text-waiting.fa-circle',
-                          'refused': 'text-error.fa-circle'
-                      }
-                  };
+            checked = (contribution) => {
```

Comment:
We haven't actually decided on this ES2015 code style, but I think we should always use the short form of arrow-functions when you only have the return statement inside the function block. Like:

`checked = (contribution) => _.contains(args.selectedContributions(), contribution.id);`